app/main.py

```diff
@@ -1 +1,16 @@
-# write your code here
+from app.cafe import Cafe
+from app.errors import VaccineError, NotWearingMaskError
+
+
+def go_to_cafe(friends: list, cafe: Cafe) -> str:
```

Comment:
Code Style Issue: It's a good practice to use type annotations for all function parameters and return values. Here, the type of `cafe` is not annotated.